_data/simple-icons.json

```
@@ -10969,6 +10969,11 @@
                 "type": "MIT"
             }
         },
+        {
+            "title": "V2EX",
+            "hex": "222223",
```

```suggestion
            "hex": "1F1F1F",
```

The `#1F1F1F` is picked from the darkest gradient value in `icon-192.png`.